18F/identity-idp
# LG-6159: Add icons for personal key buttons

**Why**: For feature parity with the existing personal key screen experience.

**Testing Instructions:**

1. Set `idv_api_enabled: "true"` in local `config/application.yml`
2. Go to: http://localhost:3000/verify/v2/personal_key
3. Observe icons for each of the three buttons "Download", "Print", "Copy"

**Screenshots:**

Before|After
---|---
![localhost_3000_verify_v2_personal_key (3)](https://user-images.githubusercontent.com/1779930/163622659-b93e07e0-77f8-4d92-bcd7-ba94e4963978.png)|![localhost_3000_verify_v2_personal_key (2)](https://user-images.githubusercontent.com/1779930/163622665-8e85d98c-418a-4e59-a3b6-dab4b0945494.png)
